pnpm-lock.yaml

```diff
@@ -7527,6 +7527,7 @@ packages:
     engines: {node: '>= 12.0.0'}
     dependencies:
       detect-libc: 1.0.3
+      napi-wasm: 1.1.0
```

Comment:
I guess it won't break anything, but this diff is interesting. Since the version is same, won't the dependencies be same? 
https://github.com/parcel-bundler/lightningcss/blob/d44a73c3c000ccebbe8355b7cf9272236e573b4d/package.json#L41-L43
lightningcss@1.21.5 doesn't have `napi-wasm` as a dependency. 🤔

Reply:
Uh... yes, I'll remove this diff. We can get it as part of updating deps.